src/Avalonia.Controls.ColorPicker/ColorSpectrum/ColorSpectrum.cs

```diff
@@ -1187,7 +1171,7 @@ private async void CreateBitmapsAndColorMap()
                 }
             });
 
-            Dispatcher.UIThread.Post(() =>
+            await Dispatcher.UIThread.InvokeAsync(() =>
```

Comment:
Why switch from Post to InvokeAsync? The original code didn't have any await and was done completely async and the bitmaps came in a bit later after the control loads.

Reply:
This method is a scope for pooled arrays. Without awaiting for Dispatcher call, pooled arrays are going to be disposed way too early.

Reply:
Hm, I need to test this then. As long as we aren't blocking loading everything else and displaying the control awaiting the bitmaps to load it's fine. But bitmaps must be completely async -- especially on WASM they take a long time to load.